### PR TITLE
Examples: Fix style issue in flow.

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: -

**Description**

This PR fixes a bug and a few minor style issues in `flow`.

### CODE REVIEW

File: examples/jsm/libs/flow.module.js [hunk 1]
```diff
@@ -1452,7 +1452,7 @@ class Node extends Serializer {
 
 	serialize( data ) {
 
-		const { x, y, style } = this.getPosition();
```
Comment (Mugen87) [inline examples/jsm/libs/flow.module.js:1455]: @sunag The object returned by `getPosition()` does not hold a `style` property. 

It seems more correct to serialize the value of `this.style`.

**Comment by marcofugaro** [comment]
I think @sunag is copying over that file from here: https://github.com/sunag/flow/blob/master/build/flow.module.js

Maybe it's better to edit the source of the lib to avoid it being replaced next time..

**Comment by Mugen87** [comment]
@sunag Should I open a PR at your repository? Or do you upstream the PR this time?

**Comment by sunag** [comment]
@Mugen87 You can merge it, I sync this in some another update.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**